src/index.ts

```diff
@@ -1,40 +1,18 @@
-import AsyncAirtable = require("asyncairtable");
-import { AirtableRecord, QueryObject } from "asyncairtable/lib/@types";
-
-const asyncAirtable = new AsyncAirtable("keyyCSADD61J0D4wQ", "appGzEQkD3QySIDvj", { baseURL: "https://api.vacfind.org/v0" });//, { ...CONFIG }
-
-
-export const helloWorld = () => {
-	console.log("Hello World");
-}
-
-export var loadLinks = async (filterOpts:QueryObject) => {
-
-	// let allrecords = []
-
-	return asyncAirtable.select('links', {
-		sort: [
-			{ field: 'group', direction: 'asc' }
-		],
-		where: filterOpts
-	}).then((results: AirtableRecord[]) => persistToStorage(results))
```

Comment:
Given that this is a library, I agree that the caching should not be a default behavior in case this code is not used in a browser. However, since our site is currently making use of this behavior as a fallback in case the API or Airtable has issues, I'd prefer not to just totally rip this functionality out.

I thought of a few alternative options and was curious which you think would be best:
1. move this behavior to a separate class that is responsible for handling the persisting of data. This way the client side would be responsible for grabbing the data with `getLinks` and passing it through this persistence class to persist the data if needed
2. add a parameter to `getLinks` that allows the client to disable the persistence in case they use this library on something other than a browser

Reply:
I believe :

1) would be a better option if this persistence method is just an optional feature and also if we'd like to maintain the " separation of concerns" principle. Not to mix the database access logic with optional functionality logic. 

2) would be better, if we'd like to encourage the user to use the persistence feature every single time before/after data fetching. Users with no browser are required to toggle off the functionality manually with the said extra parameter.

Reply:
lets go with 1. (moving it to a separate class). even though this library is just for VacFind currently, i like the ability to reuse it for different data if nore databases are ever added to this lib (i.e. to keep it reusable)